src/web/components/ClientSideTokenGeneration/CstgDomainHelper.spec.tsx

```diff
@@ -19,4 +19,66 @@ describe('test domain helper functions', () => {
       expect(isValidDomain(stringList)).toEqual(true);
     });
   });
+
+  describe('filter correct paged domains', () => {
+    const domainsList = [
+      'test.com',
+      'test2.com',
+      'test3.com',
+      'test4.com',
+      'test5.com',
+      'test6.com',
+      'test7.com',
+      'test8.com',
+      'test9.com',
+      'test10.com',
+      'test11.com',
+      'test12.com',
+      'test13.com',
+      'test14.com',
+      'test15.com',
+      'test16.com',
+      'test17.com',
+      'test18.com',
+      'test19.com',
+      'test20.com',
+      'test21.com',
+      'test22.com',
+      'test23.com',
+      'test24.com',
+      'test25.com',
+      'test26.com',
+      'test27.com',
+      'test28.com',
+      'test29.com',
+    ];
+    it('should return a filtered first page domain list', () => {
+      expect(getPagedDomains(domainsList, 1, 10)).toEqual([
```

Comment:
nit: it'd be nice if we could extract these numbers into a variable so it's clearer what it is testing without having to look at the signature of the method, so something like:
```
const pageNumber = 1;
const rowsPerPage = 10;
expect(getPagedDomains(domainsList, pageNumber, rowsPerPage)).toEqual([
...
```

Then similar for the other tests.

Reply:
Done